update-component-version/Dockerfile

```diff
@@ -1,12 +1,11 @@
-FROM python:3.7-alpine3.10
+FROM python:3.9-alpine3.18
 
 # install dependencies
-RUN apk add --no-cache git bash && \
-    apk add --no-cache --virtual .build-deps gcc g++ make && \
-    wget -O /usr/bin/yq "https://github.com/mikefarah/yq/releases/download/3.1.1/yq_linux_amd64" && \
-    chmod a+x /usr/bin/yq && \
-    pip install -U pip chartpress==0.3.2 six==1.16.0 && \
-    apk del .build-deps
+RUN apk add --no-cache git bash yq && \
+    pip install -U pip \
+    chartpress==1.0.0 \
```

Comment:
are you sure this doesn't break anything?

Reply:
nope, I'm not.
I could describe my position as "cautiously optimistic" 🤣

Reply:
this changes the tag format afaik - the rest of the automation used to be pretty sensitive to that, but I'm not sure that's the case anymore.

Reply:
I have tested this and still works as expected.
**HOWEVER**: I really cannot understand for the life of my what is the purpose.
The relevant code is this:
```bash
if [[ $GITHUB_REF =~ "tags" ]]
then
  CHART_TAG="--tag $(echo ${GITHUB_REF} | cut -d/ -f3)"
fi

# [...] other vars

# build this chart to get the version
chartpress --skip-build $CHART_TAG
CHART_VERSION=$(yq eval '.version' helm-chart/${CHART_NAME}/Chart.yaml)
```

So, every time that this script is invoked and `GITHUB_REF` contains "tags", which is whenever this action is invoked upon the **release** of a Renku component, then `CHART_VERSION==CHART_TAG` and in other words building with `chartpress` is useless. The only reason for the existence of the above code is **if** this action is ever triggered by a commit that is not a tag but:
1. I don't see why would we need that
2. No Renku-components repos seems to use it that way

IMO we should just make sure that this action is called from a tag trigger and remove chartpress from the electric boogaloo...